### ISSUE TITLE
Typo in declared prop type issue

### ISSUE THREAD

**Issue by chrisahardie** [yannickcr/eslint-plugin-react#1877]
I am using React Router,  which has a ReactRouterPropTypes package for defining prop types. When using it like:

User.propTypes = {
  match: ReactRouterPropTypes.match.isRequired,
  history: ReactRouterPropTypes.history.isRequired,

I get these errors:

 >493:33  error  Typo in declared prop type: history  react/no-typos
 >494:31  error  Typo in declared prop type: match    react/no-typos

Dependencies:

```
    "eslint": "4.19.0",
    "eslint-config-airbnb": "16.1.0",
    "eslint-config-airbnb-base": "12.1.0",
    "eslint-plugin-import": "2.11.0",
    "eslint-plugin-jsx-a11y": "6.0.3",
    "eslint-plugin-react": "7.7.0",
```

**Comment by ljharb** [comment]
Indeed; typos should only be checked on React.PropTypes and the prop-types package.

**Comment by jomasti** [comment]
I believe this was fixed in v7.8.0 with #1671. I would suggest upgrading and re-verifying.
